CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - _Experimental_: Do not wrap comment nodes in `@layer` when running codemods ([#14517](https://github.com/tailwindlabs/tailwindcss/pull/14517))
 - _Experimental_: Ensure we don't lose selectors when running codemods ([#14518](https://github.com/tailwindlabs/tailwindcss/pull/14518))
 - _Experimental_: inject `@import` in a more expected location when running codemods ([#14536](https://github.com/tailwindlabs/tailwindcss/pull/14536))
+- Improve `@tailwindcss/postcss` performance for initial builds ([#14565](https://github.com/tailwindlabs/tailwindcss/pull/14565))
```

Comment:
Maybe this should be more specifc?

```suggestion
- Only setup a single compiler in `@tailwindcss/postcss` for initial builds ([#14565](https://github.com/tailwindlabs/tailwindcss/pull/14565))
```

Also should move it above the experimental stuff maybe to keep things organized.

Reply:
Yeah that's a bit better. I move it locally already, but saw CI fail so wanted to fix that first haha. Thanks for this suggestion 💪